include/z64.h

```diff
@@ -1014,10 +1014,10 @@ typedef struct {
 } struct_800BD888_arg1; // size = 0x28
 
 typedef struct {
-    /* 0x0 */ u32 type;
-    /* 0x4 */ u32 setScissor;
-    /* 0x8 */ Color_RGBA8 color;
-    /* 0xC */ Color_RGBA8 envColor;
+    /* 0x00 */ u32 type;
+    /* 0x04 */ u32 setScissor;
+    /* 0x08 */ Color_RGBA8_u32 color;
+    /* 0x0C */ Color_RGBA8 envColor;
 } struct_801F8010; // size = 0x10
```

Comment:
The leading 0's can go. I think we agreed on a style for consistency to have the number of 0's be based on the struct members, not the total struct size